packages/react/src/components/Slider/useSlider.ts

```diff
@@ -86,9 +97,18 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
     props.defaultValue,
     (ev: React.FormEvent<HTMLElement> | undefined, v: ISliderProps['value']) => props.onChange && props.onChange(v!),
   );
+  const [unclampedLowerValue, setLowerValue] = useControllableValue(
+    props.lowerValue,
+    props.defaultLowerValue,
+    (ev: React.FormEvent<HTMLElement> | undefined, v: ISliderProps['value']) => props.onChange && props.onChange(v!),
```

Comment:
Thinking more about that... it's potentially confusing/unexpected that `onChange` doesn't get called when the lower value changes. For backwards compatibility, we can't change the first param to `onChange` (which has to be `value`), but we could add a second optional `range` param that existing callers would just ignore.

`onChange: (value: number, range?: [number, number]) => void;`

And then this could become:
```suggestion
    (ev: React.FormEvent<HTMLElement> | undefined, lv: ISliderProps['lowerValue']) =>
      props.onChange?.(value, [lv!, value]),
```

Line 98 above for when `value` changes would be similar, but only pass in the second property if `props.ranged` is true:
```ts
    (ev: React.FormEvent<HTMLElement> | undefined, v: ISliderProps['value']) =>
      props.onChange?.(v!, props.ranged ? [lowerValue, v!] : undefined),
```

Reply:
The second param to `onChange` could replace the `onRangeChange` callback entirely.